src/main/java/de/komoot/photon/query/PhotonRequestFactory.java

```diff
@@ -42,13 +53,25 @@ public PhotonRequestFactory(Set<String> supportedLanguages) {
         } catch (Exception nfe) {
             //ignore
         }
+        Boolean locationDistanceSort = true;
```

Comment:
I'd suggest to switch to 'false' as default. While I like the distance sort (=stronger bias) when zoomed to greater zoom levels, the former results are closer to my expectations when zoomed out. E.g. for [Stuttgart with a location bias of (51;9)](http://46.4.67.134:2322/api?q=Stuttgart&lang=de&limit=5&lat=51&lon=9) I would expect to see Stuttgart in the results, rather than 'Stuttgarter Straße' in Frankfurt or Kassel. Perhaps providing a zoom level or weighting factor for the bias would be an option for a later PR.

Reply:
Hmmh, probably we should argue then how it was before (I think it was kind of broken ie it should be indeed false) because if you argue on expectation I can give you a counter argument like when you search for "bahnhof" I would expect the train station within that city.

> Perhaps providing a zoom level or weighting factor for the bias would be an option for a later PR.

Related to #198 ... probably we could also provide a bounding box filter.